pyproject.toml

```diff
@@ -16,6 +16,7 @@ fasttrack = 'backport.fasttrack:fasttrack'
 walkthrough = 'walkthrough.cli:cli'
 run_python_step = 'etl.run_python_step:main'
 compare = 'etl.compare:cli'
+etl-match-variables = 'etl.match_variables_from_two_versions_of_a_dataset:main_cli'
```

Comment:
Should we use a shorter name, at least for the etl command? Maybe just `match_variables`.

Reply:
Regarding the name for the command, there tare two things:

1. I have a weak preference for separator `-` (dash) over `_` (underscore)
2. Prefixing it with `etl` may ensure that there is no collision with any other tool that one may have installed on their computer.

Other commands have no prefix, and I think we could consider adding some prefix to these, but this might be something to be discussed later on...